src/I18n/RelativeTimeFormatter.php

```diff
@@ -87,15 +86,15 @@ public function diffForHumans(ChronosInterface $date, ?ChronosInterface $other =
     /**
      * Format a into a relative timestring.
      *
-     * @param \DateTimeInterface $time The time instance to format.
+     * @param \Cake\I18n\Time|\Cake\I18n\FrozenTime $time The time instance to format.
      * @param array $options Array of options.
      * @return string Relative time string.
      * @see \Cake\I18n\Time::timeAgoInWords()
      */
-    public function timeAgoInWords(DateTimeInterface $time, array $options = []): string
+    public function timeAgoInWords($time, array $options = []): string
```

Comment:
This is sad to see.
Shouldnt we then just have our own interface here that combines both? To be able to typehint sth instead of allowing anything, incl. integers coming in?